options.rb

```diff
@@ -18,10 +18,9 @@ def process_option(app, option)
       4 => -> { app.create_book },
       5 => -> { app.create_rental },
       6 => -> { app.list_rentals_for_person },
-      7 => -> { puts 'Exiting the app...' }
+      7 => -> { app.save_data('./data/books.json', './data/rentals.json', './data/people.json') }
```

Comment:
- [ ] An error occurs when exiting your app as a result of an undefined method `save_data`. Please fix this bug 🐛 
![image](https://github.com/ridabensalem/School-library/assets/46839548/388f4551-7e45-4301-8310-88019830dc9f)
In my opinion, it is no longer necessary to try to save the data on exit because you already write the data each time a new record is created 👍